frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx

```diff
@@ -127,33 +145,36 @@ export const ParcelIdentificationForm: React.FC<IIdentificationProps> = ({
             nameSpace={withNameSpace('address')}
             hideStreetAddress
           />
-          <Row className="field-row">
+          <Form.Row>
```

Comment:
I like using form.row to represent a group tags displayed on a single line. This actually provides very similar functionality to the previous commit, just with a className of form-row (but not needing to provide the classname each time).

Reply:
Ah gothcya, my bad. I think I was the one who swapped these out.